modules/schema/src/main/java/org/apache/ignite/internal/schema/BinaryTupleSchema.java

```diff
@@ -289,10 +289,23 @@ public static int nullMapSize(int numElements) {
     }
 
     /**
-     * Returns the null map size in bytes if there are nullable elements, zero otherwise.
+     * Get offset of the byte that contains null-bit of a given tuple element.
+     *
+     * @param index Tuple element index.
+     * @return Offset of the required byte relative to the tuple start.
+     */
+    public static int getNullOffset(int index) {
```

Comment:
lets rename this to `nullByteOffset`, because that name is better reflects the essence

Reply:
Perhaps the current name is not very good. But I think `nullByteOffset` would be confusing. People could misread it as "Offset of a null byte" and wrongfully think that this is about some byte with zero value. So to be entirely clear it should be something like `byteOffsetOfNullBitOctet`. What do you think?